util/libcrypto.num

```diff
@@ -4597,3 +4597,5 @@ EVP_MAC_do_all                          4550	1_1_2	EXIST::FUNCTION:
 EVP_MAC_do_all_sorted                   4551	1_1_2	EXIST::FUNCTION:
 EVP_str2ctrl                            4552	1_1_2	EXIST::FUNCTION:
 EVP_hex2ctrl                            4553	1_1_2	EXIST::FUNCTION:
+EVP_PKEY_supports_digest_nid            4554	1_1_2	EXIST::FUNCTION:
+
```

Comment:
Remove the empty line.  That's what's causing the Travis failure.

Reply:
Sorry. On vacation this week, a bit rushed while being frowned at for looking at the computer so I didn't let `make test` complete. Fixed now.

Reply:
That particular line doesn't trigger any test, but it does trigger `make update`